protocol/src/lib.rs

```diff
@@ -148,16 +149,22 @@ pub enum Message {
      */
     /// Send a close the given extent ID on the downstairs.
     /// We send the downstairs the repair ID (rep_id) and the extent number.
-    ExtentClose(u64, u64),
-    /// Ack the close of an extent from the downstairs using the rep_id.
-    ExtentCloseAck(u64),
+    ExtentClose(u64, usize),
     /// Send a request (with rep_id) to reopen the given extent.
-    ExtentReopen(u64, u64),
```

Comment:
why not u64 for extent id?

Reply:
It's a usize on the downstairs side (as it is an index into the vec of extents) and we have to convert it
when it arrives, so I was thinking why not just keep it usize everywhere.  So I changed it here and
in the place where we determine what extents need repair so it's just always usize